package.json

```diff
@@ -1,46 +1,22 @@
 {
-  "name": "@netlify/hydrogen-platform",
+  "name": "@netlify/hydrogen-platform-ws",
```

Comment:
Is the `-ws` mean to denote website since there is also `@netlify/hydrogen-platform` under `packages/platform` now?